miso-web/src/main/webapp/scripts/hot_dilution.js

```diff
@@ -0,0 +1,128 @@
+          include : true,
+          unpack : function(dil, flat, setCellMeta) {
+            flat.libraryAlias = dil.library.alias;
+          },
+          pack : function(dil, flat, errorHandler) {
+          }
+        },
+        HotUtils.makeColumnForFloat('Conc. (' + Constants.libraryDilutionConcentrationUnits + ')', true, 'concentration'),
+        {
+          header : 'Creation Date',
+          data : 'creationDate',
+          type : 'date',
+          dateFormat : 'YYYY-MM-DD',
+          datePickerConfig : {
+            firstDay : 0,
+            numberOfMonths : 1
+          },
+          allowEmpty : false,
+          validator : HotUtils.validator.requiredText,
+          include : true,
+          unpack : function(dil, flat, setCellMeta) {
+            flat.creationDate = dil.creationDate;
+          },
+          pack : function(dil, flat, errorHandler) {
+            dil.creationDate = flat.creationDate;
+          }
+        },
+        {
+          header : 'Targeted Sequencing',
+          data : 'targetedSequencingAlias',
+          type : 'dropdown',
+          trimDropdown : false,
+          source : [],
+          validator : HotUtils.validator.permitEmpty,
+          include : Constants.isDetailedSample,
+          unpack : function(dil, flat, setCellMeta) {
+            flat.targetedSequencingAlias = Utils.array.maybeGetProperty(
+                Utils.array.findFirstOrNull(Utils.array
+                    .idPredicate(dil.targetedSequencingId),
+                    Constants.targetedSequencings), 'alias') || '(None)';
+          },
+          pack : function(dil, flat, errorHandler) {
+            dil.targetedSequencingId = Utils.array
+                .maybeGetProperty(
+                    Utils.array
+                        .findFirstOrNull(
+                            function(tarSeq) {
+                              return flat.targetedSequencingAlias == tarSeq.alias && tarSeq.kitDescriptors
+                                  .indexOf(dil.library.kitDescriptorId) != -1;
+                            }, Constants.targetedSequencings), 'id');
+          },
+          depends : '*start', // This is a dummy value that gets this run on
+          // creation only
+          update : function(dil, flat, value, setReadOnly, setOptions, setData) {
+            setOptions([ '(None)' ].concat(Constants.targetedSequencings
+                .filter(
+                    function(targetedSequencing) {
+                      return targetedSequencing.kitDescriptors
+                          .indexOf(dil.library.kitDescriptorId) != -1;
+                    }).map(Utils.array.getAlias).sort()));
+          }
+        } ];
+  },
+  
+  bulkActions : [
+      {
+        name : 'Edit',
+        action : function(ids) {
+          window.location = window.location.origin + '/miso/library/dilution/bulk/edit/' + ids
+              .join(',');
+        }
+      },
+      {
+        name : 'Create one pool',
```

Comment:
I don't know what to call this one (or the next one). “Create one pool per dilution” is really long for a button.